EdFi.Buzz.Etl/eng/edfi.buzz.etl.nuspec

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package>
+  <metadata>
+    <id>edfi.buzz.etl</id>
+    <version>$version$</version>
+    <title>Ed-Fi Buzz ETL</title>
+    <authors>Ed-Fi Alliance, LLC and contributors</authors>
+    <projectUrl>https://github.com/Ed-Fi-Alliance-OSS/Buzz</projectUrl>
+    <copyright>Copyright © 2020, Ed-Fi Alliance, LLC and contributors.</copyright>
+    <license type="expression">Apache-2.0</license>
+    <summary>Buzz survey loader</summary>
+    <description>Loads Buzz student surveys files</description>
+  </metadata>
+  <files>
+    <file src=".\Windows\*" target="windows" />
+    <!-- NB: nuget.exe does not handle widlcards with forward-slash paths. Must use back-slash. -->
+    <file src="..\dist\*.json" target="dist" />
+    <file src="..\dist\config\**" target="dist/config" />
+    <file src="..\dist\processors\**" target="dist/processors" />
+    <file src="..\dist\sql\**" target="dist/src" />
+    <file src="..\dist\src\**" target="dist/src" />
+    <file src="..\dist\tasks\**" target="dist/tasks" />
```

Comment:
Do these need to be called out separately, or will it work with just 

```
<file src="..\dist" target="dist/" />
```

Now that I'm thinking about it, I think that ended up either not being recursive or it dumped everything into the same folder. Can you confirm?

Reply:
We can use the directory file listing. It does create the dist folder with the files in their proper folders.